internal/ingress/controller/template/template_test.go

```diff
@@ -226,12 +226,12 @@ func TestBuildLuaSharedDictionaries(t *testing.T) {
 func TestLuaConfigurationRequestBodySize(t *testing.T) {
 	cfg := config.Configuration{
 		LuaSharedDicts: map[string]int{
-			"configuration_data": 10, "certificate_data": 20,
+			"configuration_data": 10000, "certificate_data": 20000,
 		},
 	}
 
 	size := luaConfigurationRequestBodySize(cfg)
-	if size != "21" {
+	if size != "21M" {
```

Comment:
this seems wrong to me (with the below message) 

t.Errorf("expected the size to be 20 but got: %v", size) vs if size != "21M" :)

Reply:
idek why it was 20 before, but I'll add the "M" now, ty!